src/js/tabs/auxiliary.js

```diff
@@ -1,8 +1,13 @@
 import { i18n } from '../localization';
 import GUI from '../gui';
 import { get as getConfig, set as setConfig } from '../ConfigStorage';
-import { bit_check } from '../serial_backend';
+import { bit_check } from '../bit';
 import { mspHelper } from '../msp/MSPHelper';
+import FC from '../FC';
```

Comment:
```suggestion
import FC from '../fc';
```

Reply:
🤦‍♂️ Thanks macOS.

Reply:
The messed up thing is that it would load in the module just fine, but for some reason it would be different instance of that module. 😭 Anyways thanks a lot